src/components/Card/Subscription/SubscriptionCard.test.tsx

```diff
@@ -1,68 +1,10 @@
 import React from 'react';
 import {shallow, ShallowWrapper} from 'enzyme';
 
-import SubscriptionCard from '.';
-
-// Mock functions
-const editHandler = jest.fn();
-const deleteHandler = jest.fn();
-
-// Mock data for cardBody props
-const cardBody = {
-    list: ['List Item 1', 'List Item 2', 'List Item 3'],
-    labelValuePairs: [
-        {label: 'Label 1', value: 'Value 1'},
-        {label: 'Label 2', value: 'Value 2'},
-        {label: 'Label 3', value: 'Value 3'},
-        {label: 'Label 4', value: 'Value 4'},
-    ],
-};
+// Mock data
+import { subscriptionCardBody as cardBody, subscriptionCardProps } from '@Constants/mockData/card';
```

Comment:
```suggestion
import {subscriptionCardBody as cardBody, subscriptionCardProps} from '@Constants/mockData/card';
```
There should also be an eslint rule for this.

Reply:
This was a lint error, there is already a rule for this.